### PR TITLE
Approval Letter in Resources #1270

### PR DESCRIPTION
**Pull request by peeyush14goyal**
Issue #1270
Approval Letter in Resources

Added the Kerala Logo image in public/images folder
If assigned_facility_object is NULL then 
image will not be shown

Optimized for Mobile and tablet View

Screenshots
image
image

### CODE REVIEW

**Comment by aparnacoronasafe** [comment]
@bodhish Preview deploy failed. Please merge to staging and add a label "test in staging"

**Comment by bodhish** [comment]
@aparnacoronasafe @peeyush14goyal I have triggered rebuild for Netlify

**Comment by peeyush14goyal** [comment]
@bodhish Great. Preview is working

**Review by smartclash — APPROVED** [review]

File: src/Components/Resource/ResourceDetails.tsx [hunk 1]
```diff
@@ -93,220 +97,356 @@ export default function ResourceDetails(props: { id: string }) {
...
-  if (isLoading) {
-    return <Loading />;
-  }
-
-  return (
-    <div>
-      <div className="mx-3 md:mx-8 mb-10">
-        <div className="my-4 flex justify-between items-center">
-          <PageTitle title={"Resource details"} />
+  const printData = (data: any) => {
```
Comment (ghost) [inline src/Components/Resource/ResourceDetails.tsx]: `printData` is not suitable name for approval letter view. :sweat_smile:. Name it something else like below
```suggestion
  const ApprovalLetter = (data: any) => {
```

File: src/Components/Resource/ResourceDetails.tsx [hunk 2]
```diff
@@ -93,220 +97,356 @@ export default function ResourceDetails(props: { id: string }) {
...
-          <div>
-            <h4 className="mt-8">Orgin Facility</h4>
-
-            {showFacilityCard(data.orgin_facility_object)}
-          </div>
-          <div>
-            <h4 className="mt-8">Resource Approving facility</h4>
+          <div className="grid md:grid-cols-3 gap-4 pb-4">
+            <div>
+              <h4 className="mt-8">Orgin Facility</h4>
```
Comment (ghost) [inline src/Components/Resource/ResourceDetails.tsx]: Is the spelling correct?

**Review by abhinandanarya06 — APPROVED** [review]
Fix the suggestions. 
Remaining all looks good to me.

File: src/Components/Resource/ResourceDetails.tsx (hunk 2 above)
Comment (bodhish) [reply src/Components/Resource/ResourceDetails.tsx]: No

Comment (peeyush14goyal) [reply src/Components/Resource/ResourceDetails.tsx]: In API also its same so I added that only

**Comment by peeyush14goyal** [comment]
@abhinandanarya06 Nicely done  ✔

**Comment by aparnacoronasafe** [comment]
teste failed. 2 bugs discovered

1- the content of the letter must say if the request is approved or not. additionally if you intend to communicate the exact status of the request it must say:

" The request for a resource (details below) placed by yourself is APPROVED and the status of the request "TRANSFER IN PROGRESS" 

currently, the sentence that appears there does not make complete sense. 
Screenshot 2021-06-05 at 11 49 18 PM


2) when the approved quantity id entered while approval, still approved quantity in the letter appears as zero

**Comment by peeyush14goyal** [comment]
For bug 2
@aparnacoronasafe I am showing that parameter which is shown in Resource Details Section. Please confirm if the data shown in **Approved Quantity field** in Resource Details is also wrong
image

**Comment by peeyush14goyal** [comment]
@vigneshhari Can you tell me possible values of the status field of Resource Details?

**Comment by peeyush14goyal** [comment]
@aparnacoronasafe I have made some changes. Please check.

**Comment by vigneshhari** [comment]
Have you checked swagger to see the possible Values ? It should be available there.

**Comment by peeyush14goyal** [comment]
@aparnacoronasafe  Bug 1 is fixed

**Comment by peeyush14goyal** [comment]
@vigneshhari For approving quantity we have `assigned_quantity` field. Right? Coz I see that's what is used in the resources

**Comment by aparnacoronasafe** [comment]
tested in preview

**Pull request merged**